modules/configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery_{context}"]
+= Setting up a private storage endpoint on Microsoft Azure by configuring the Image Registry Operator to discover VNet and subnet names
+
+The following procedure shows you how to set up a private storage endpoint on Microsoft Azure by configuring the Image Registry Operator to discover VNet and subnet names.
+
+.Prerequisites 
+
+* You have configured the image registry to run on Azure.
+* Your network has been set up using the Installer Provisioned Infrastructure (IPI) installation method. 
++
+For users with a custom network setup, see "Configuring a private storage endpoint on Azure with user-provided VNet and subnet names". 
+
+.Procedure 
+
+. Edit the Image Registry Operator `config` object and set `networkAccess.type` to `Internal`:
++
+[source,terminal]
+----
+$ oc edit configs.imageregistry/cluster
+----
++
+[source,terminal]
+----
+# ...
+spec:
+  [...]
+   storage:
+      azure:
+        [...]
+        networkAccess:
+          type: Internal
+# ...
+----
+
+. Optional. Enter the following command to confirm that the Operator has completed provisioning. This might take a few minutes. 
```

Comment:
```suggestion
. Optional: Enter the following command to confirm that the Operator has completed provisioning. This might take a few minutes. 
```

I won't point out any more instances of this, if you wouldn't mind checking throughout the rest of the PR

Reply:
Yep. There was only 2. Thanks!